File: front-end/GServices/src/app/models/conges.ts
import { Personnel } from "./personnel";

export class Conges{
    idConges:number;
    dureeD:string;
    dureeF:string;
    raisons:string;
    personnel:Personnel;
    constructor(dureeD:string,dureeF:string,raisons:string,personnel:Personnel) {
        this.dureeD=dureeD;
        this.dureeF=dureeF;
        this.raisons=raisons;
        this.personnel=personnel;
    
   
}}

File: back-end/Services/src/main/java/com/angular/Services/entities/Conges.java
package com.angular.Services.entities;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Builder
@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
public class Conges {
    @Id
    @GeneratedValue
    private Long idConges;
    private String dureeD;
    private String dureeF;

    private String raisons;
    @ManyToOne
    @JoinColumn(name="idPersonnel")
    private Personnels personnel;



}
